### DISCUSSION TITLE
gh: Not Found (HTTP 404)     "message": "Not Found",   "documentation_url": "https://docs.github.com/rest/collaborators/collaborators#add-a-repository-collaborator

### DISCUSSION THREAD

**Discussion by Shreenivasayg** [community/community#34623] [Apps, API and Webhooks]
I am trying to add collaborators through GitHub CLI. Done with gh auth login. I am using following cli in terminal. i have  admin privilege's over this private repo. 
gh api \
  --method PUT \
  -H "Accept: application/vnd.github+json" \
  /repos/OWNER/REPO/collaborators/USERNAME \
  -f permission='triage'
(Adding repo ownername, repo name, and username as required) example
gh api \
  --method PUT \
  -H "Accept: application/vnd.github+json" \
  /repos/rohit/services/collaborators/sravanthi \
  -f permission='triage'
Output
Error
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/collaborators/collaborators#add-a-repository-collaborator"
}
gh: Not Found (HTTP 404)

**Comment by antonblr** [comment]
@Shreenivasayg - do you have access to https://github.com/rohit/services repo ? It is `404` for me, but perhaps it is private - in this case you'd need owner to grant you permissions for it already.

**Reply by Shreenivasayg** [reply]
I admin access to it, and rohit is ownername , should it be org name?

**Reply by antonblr** [reply]
@Shreenivasayg - if the url above work for you, that means that user `rohit` owns it. Not sure what org name you meant.

Check `gh` cli is authorized with correct credentials then, run:
```
$ gh auth status
```
Also do a sanity check to make sure that overall `gh` cli permissions are correct and you have basic access to the repo:
```
$ gh repo view https://github.com/rohit/services

# or with api command e.g.
$ gh api /repos/rohit/services/commits
```
^ if you cannot - `gh` cli auth is not setup correctly.

**Reply by Shreenivasayg** [reply]
gh auth status    showing command not found . 
i had done gh auth login>github.com>https>PAT

**Reply by Shreenivasayg** [reply]
may i know the complete  procedure to set up

**Reply by antonblr** [reply]
I thin you've done setup already - https://cli.github.com/manual/gh_auth_login. Just make sure your PAT have **enough permissions**.

`gh auth status` should work - https://cli.github.com/manual/gh_auth_status. Check `gh` version and update if it is old.

**Reply by Shreenivasayg** [reply]
@antonblr Checked both requirements. Both are done properly. Still same error.

**Reply by Shreenivasayg** [reply]
I am getting responses for  $ gh repo view https://github.com/rohit/services
 or with api command e.g.
$ gh api /repos/rohit/services/commits 
But still cannot add new  collaborators

**Comment by KodeyThomas** [comment]
Services doesn't seem to exist! Unless it's a private repo. Minus that looks all good

**Reply by Shreenivasayg** [reply]
There is Private repo in that name and i have admin access

**Reply by Shreenivasayg** [reply]
What Services you are refereeing to?
